variables.tf

```diff
@@ -1,11 +1,67 @@
+variable "bucket_name" {
+  default     = null
+  description = "Name to use for bucket (optionally leave blank and use `bucket_suffix`)"
+  type        = string
+}
+
+variable "bucket_suffix" {
+  default     = "bucket"
+  description = "Suffix to apply to bucket, which will be created in format [ACCOUNT_ID]-[REGION]-cloudberry-[BUCKET_SUFFIX]. This is not used if `bucket_name` is set to a non-null value."
+  type        = string
+}
+
+variable "cloudberry_account" {
+  default     = "626709717326"
```

Comment:
Who's account is this? I'm guessing we don't want PII in the PR

Reply:
Account IDs aren't sensitive, and this one is their account, published in their docs. Actually including it is the point, that's something that would help someone who wants to implement this bucket.